backend/app/services/flow_scheduler.py

```
@@ -0,0 +1,389 @@
+
+                            for flow in due_flows:
+                                execute_flow(db, flow)
+                        else:
+                            logger.debug("[flow_scheduler] No flows due for execution")
+
+                    finally:
+                        db.close()
+
+                    logger.debug("[flow_scheduler] Scheduler cycle completed")
+
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)